api/v1beta1/flinkcluster_types.go

```diff
@@ -675,6 +693,7 @@ type FlinkClusterStatus struct {
 
 // +kubebuilder:object:root=true
 // +kubebuilder:subresource:status
+// +kubebuilder:subresource:scale:specpath=.spec.taskManager.replicas,statuspath=.status.components.taskManagerStatefulSet.replicas,selectorpath=.status.components.taskManagerStatefulSet.selector
```

Comment:
I don't quite understand the meaning and purpose of this scale subresource, could you add a comment?

Reply:
Adding